platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -264,6 +264,34 @@ sep1:
     #
     value: |
 
+sep6:
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-6 for this instance of AP server.
+  #
+  enabled: false
+  ## @param: transactions
+  ## Configures the GET transactions endpoint
+  #
+  transactions:
+    ## @param: enabled
+    ## Whether to enable the GET transactions endpoint
+    #
+    enabled: true
+  ## @param: transaction
+  ## Configures the GET transaction endpoint
+  #
+  transaction:
```

Comment:
We should not configure whether to enable or disable the `GET /transaction` endpoint because it is required by SEP-6.